ReactiveCocoa/Swift/Property.swift

```diff
@@ -581,6 +593,8 @@ public final class MutableProperty<Value>: MutablePropertyProtocol {
 	///   - initialValue: Starting value for the mutable property.
 	public init(_ initialValue: Value) {
 		(signal, observer) = Signal.pipe()
+		token = Lifetime.Token()
+		lifetime = Lifetime(token)
```

Comment:
Every stored property is initialised in the initializer, so I am just following the pattern.

Besides, `lifetime` needs the `token`, which I don't think it can be expressed inline as `self` cannot be used, unless we make it `lazy` or something like `(..., ...) = Lifetime.create()` (but then we cannot declare different access levels...).